modules/nested-tabs/widgets/nested-tabs.php

```diff
@@ -187,8 +187,8 @@ protected function register_controls() {
 			],
 			'separator' => 'before',
 			'selectors_dictionary' => [
-				'top' => '--n-tabs-direction: column; --n-tabs-heading-direction: row; --n-tabs-heading-width: initial;',
-				'bottom' => '--n-tabs-direction: column-reverse; --n-tabs-heading-direction: row; --n-tabs-heading-width: initial;',
+				'top' => '--n-tabs-direction: column; --n-tabs-heading-direction: row; --n-tabs-heading-width: initial; --n-tabs-title-flex-basis: content; --n-tabs-title-flex-shrink: 0;',
+				'bottom' => '--n-tabs-direction: column-reverse; --n-tabs-heading-direction: row; --n-tabs-heading-width: initial; --n-tabs-title-flex-basis: content; --n-tabs-title-flex-shrink: 0',
```

Comment:
Hi @mykytamurzin ,

I think you should define all values for all options.

Reasoning: it should be fully responsive and one breakpoint should be able to overwrite an other value.

For e.g. end and start possibly you can use 'initial' as value.